src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private final int maxLimit = 100;

```

Comment:
```suggestion
    private final int MAX_POSSIBLE_NUMBER = 100;
```

Reply:
If the name of this constant is in uppercase, command "mvn clean package" reports an error:
[ERROR] C:\java_projects\jv-lottery\src\main\java\core\basesyntax\Lottery.java:8:23: Abbreviation in name 'MAX_POSSIBLE_NUMBER' must contain no more than '2' conse
cutive capital letters. [AbbreviationAsWordInName]
[ERROR] C:\java_projects\jv-lottery\src\main\java\core\basesyntax\Lottery.java:8:23: Member name 'MAX_POSSIBLE_NUMBER' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9
]*$'. [MemberName]
Audit done.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE

What can I do to solve this?